integration_tests/run_pyspark_from_build.sh

```diff
@@ -114,7 +114,16 @@ else
           $RUN_TEST_PARAMS
           "$@")
 
+    NUM_LOCAL_EXECS=${NUM_LOCAL_EXECS:-0}
```

Comment:
documentation?

Reply:
also to clarify it might be nice to add to README

Reply:
Nice catch, this feature should be documented in integration_tests/README.md